keymaps/latex.json

```diff
@@ -1,7 +1,11 @@
 {
+  "atom-text-editor": {
+    "ctrl-alt-l": "latex:toggle-log"
+  },
   "atom-text-editor[data-grammar~=\"latex\"]": {
     "ctrl-alt-b": "latex:build",
     "ctrl-alt-s": "latex:sync",
-    "ctrl-alt-c": "latex:clean"
+    "ctrl-alt-c": "latex:clean",
+    "ctrl-alt-e": "latex:sync-log"
```

Comment:
Is there a keybinding we can use? I think is a cool feature. It basically highlights and scrolls to errors that are tied to the current line.